src/Umbraco.AuthorizedServices/Services/Implement/AuthorizationParametersBuilder.cs

```diff
@@ -6,7 +6,27 @@ internal sealed class AuthorizationParametersBuilder : IAuthorizationParametersB
 {
     public Dictionary<string, string> BuildParameters(ServiceDetail serviceDetail, string authorizationCode, string redirectUri, string codeVerifier)
     {
-        var parametersDictionary = new Dictionary<string, string>
+        var parametersDictionary = new Dictionary<string, string>();
```

Comment:
`AuthorizationParametersBuilderTests` should be extended to cover this logic.  There's currently just one test:

- BuildParameters_ReturnsExpectedResult

We should now have:

- BuildParameters_ForOAuth2AuthorizationCode_ReturnsExpectedResult
- BuildParameters_ForOAuth2AuthorizationCode_WithScopes_ReturnsExpectedResult
- BuildParameters_ForOAuth2ClientCredentials_ReturnsExpectedResult
- BuildParameters_ForOAuth2ClientCredentials_WithClientCredentialsProvisionInRequestBody_ReturnsExpectedResult

With appropriate refactoring to DRY up any common test logic.

Reply:
With appropriate naming changes if you also apply the suggestion in the comment starting _"It might be cleaner to add some method overloads here..."_

Reply:
Updated